qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -36,6 +36,40 @@ class FineAmplitudeCal(BaseCalibrationExperiment, FineAmplitude):
         experiment the circuits that are run have a custom gate with the pulse schedule attached
         to it through the calibrations.
 
+    # section: example
+        .. jupyter-execute::
+            :hide-code:
+
+            # backend
+            from qiskit_experiments.test.pulse_backend import SingleTransmonTestBackend
+            backend = SingleTransmonTestBackend(5.2e9,-.25e9, 1e9, 0.8e9, 1e4, noise=False, seed=101)
+
+        .. jupyter-execute::
+
+            import numpy as np
+            from qiskit.circuit.library import XGate
+            from qiskit_experiments.calibration_management.calibrations import Calibrations
+            from qiskit_experiments.calibration_management.basis_gate_library \
+            import FixedFrequencyTransmon
+            from qiskit_experiments.library import FineAmplitudeCal
+
+            library = FixedFrequencyTransmon()
+            cals = Calibrations.from_backend(backend=backend, libraries=[library])
+            exp_cal = FineAmplitudeCal(physical_qubits=(0,),
+                                       calibrations=cals,
+                                       schedule_name="x",
+                                       backend=backend,
+                                       cal_parameter_name="amp",
+                                       auto_update=True,
+                                       gate=XGate(),
+                                       measurement_qubits=(0,))
+            # This option is necessary!
+            exp_cal.analysis.set_options(fixed_parameters={"angle_per_gate" : np.pi,
+                                                           "phase_offset" : np.pi/2})
+
+            cal_data = exp_cal.run().block_for_results()
+            display(cal_data.figure(0))
+            cal_data.analysis_results(dataframe=True)
```

Comment:
These results are improved by choosing parameter values in qiskit_experiments.calibration_management.basis_gate_library.FixedFrequencyTransmon(). 
If you still want to remove these code examples of calibration experiments in the file fine_amplitude.py, 
tell me your opinion. I will agree to your suggestion.

Reply:
The error is a little bit large but it is better now. I think this is good enough:

![image](https://github.com/user-attachments/assets/a9cd32bd-8ba2-4684-90ee-00cde97152a7)

The `FineAmplitudeCal` example I am not as sure of. It is like the `FineXAmplitudeCal` cal but it does not have the pre-circuit so it bounces the state between 0 and 1 instead of around 0.5. That will make it less sensitive to small errors, so it is not a great example. In practice, I think you often need to subclass `FineAmplitudeCal` and change the pre-circuit. The sx cal does not need a pre-circuit so it would be a better example for `FineAmplitudeCal`.

Reply:
done.